java-checks/src/main/java/org/sonar/java/checks/DeprecatedHashAlgorithmCheck.java

```diff
@@ -44,31 +44,33 @@
   private static final String MD5 = "MD5";
   private static final String SHA1 = "SHA1";
 
-  private static final Map<String, String> ALGORITHM_BY_METHOD_NAME = ImmutableMap.<String, String>builder()
-    .put("getMd5Digest", MD5)
-    .put("getShaDigest", SHA1)
-    .put("getSha1Digest", SHA1)
-    .put("md5", MD5)
-    .put("md5Hex", MD5)
-    .put("sha1", SHA1)
-    .put("sha1Hex", SHA1)
-    .put("sha", SHA1)
-    .put("shaHex", SHA1)
-    .build();
+  private static final Map<String, String> ALGORITHM_BY_METHOD_NAME = new HashMap<>();
+    static {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)